tests/sentry/api/endpoints/test_assistant.py

```diff
@@ -27,9 +27,13 @@ def test_invalid_inputs(self):
         assert resp.status_code == 400
 
     def test_activity(self):
+        GUIDES_WITH_SEEN = GUIDES.copy()
```

Comment:
pls don't use caps for method local variables

Reply:
fixed